packages/ExpandCollapse/__tests__/ExpandCollapse.spec.jsx

```diff
@@ -216,15 +216,12 @@ describe('ExpandCollapse', () => {
     it('triggers individual panel callbacks when panels are opened and closed', () => {
       const onPanelToggle = jest.fn()
 
-      const { expandCollapse, clickPanel } = doMount(
+      const { clickPanel } = doMount(
         <ExpandCollapse>{aPanel({ id: 'panel-1', onToggle: onPanelToggle })}</ExpandCollapse>
       )
 
       clickPanel('panel-1')
       expect(onPanelToggle).toHaveBeenCalledWith(true)
-
```

Comment:
Why was this test removed? With this change, this only tests whether callbacks are called when panels are opened.